### ISSUE TITLE
Latest release (0.11.0) fails when reporterOutput is set to the default value (null)

### ISSUE THREAD

**Issue by jeffpeck-intellify** [gruntjs/grunt-contrib-jshint#256]
I have found that I need to add `reporterOutput: ""` to my `jshint` options to not get the error "Warning: Path must be a string."

The problem is in `tasks/lib/jshint.js`, here:
```
// Get reporter output directory for relative paths in reporters
    if (options.hasOwnProperty('reporterOutput')) {
      var reporterOutputDir = path.dirname(options.reporterOutput);
      delete options.reporterOutput;
    }
```

When `reporterOutput` is null, then `path.dirname()` fails.

There is a thread about it here:
https://github.com/jshint/jshint/issues/2922

**Comment by jsr6720** [comment]
Confirmed this is an issue. See related changeset here:
https://github.com/gruntjs/grunt-contrib-jshint/commit/baa5913cecbf159c5cd47be524cf5d1f7baf099c#diff-5012c18b8f548457498e70750cda5a94

**Comment by jsr6720** [comment]
Added PR #257

**Comment by XhmikosR** [comment]
Fixed by #257.
